ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-api-gnmi.j2

```diff
@@ -36,6 +36,33 @@
 {%             endfor %}
 {%         endif %}
 {%     endif %}
+
+{%     if management_api_gnmi.transport is arista.avd.defined %}
+
+| Transport | Destination | Port | gNMI SSL Profile | Tunnel SSL Profile | VRF | Local Interface | Local Port | Target ID |
+| --------- | ----------- | ---- | ---------------- | ------------------ | --- | --------------- | ---------- | --------- |
+{%         if management_api_gnmi.transport.grpc_tunnel is arista.avd.defined %}
+{%             for transport in management_api_gnmi.transport.grpc_tunnel %}
+{%                 if transport.name is arista.avd.defined %}
+{%                     set transport_name = transport.name %}
+{%                     set destination = transport.destination.address %}
+{%                     set port = transport.destination.port %}
+{%                     set gnmi_ssl_profile = transport.gnmi_ssl_profile | arista.avd.default('-') %}
+{%                     set tunnel_ssl_profile = transport.tunnel_ssl_profile | arista.avd.default('-') %}
+{%                     set vrf = transport.vrf | arista.avd.default('-') %}
+{%                     set local_interface = transport.local_interface.interface | arista.avd.default('-') %}
+{%                     set local_port = transport.local_interface.port | arista.avd.default('-') %}
+{%                     if transport.target.use_serial_number is arista.avd.defined (true) %}
+{%                         set target_id = arista.avd.default('Serial Number') %}
+{%                     elif transport.target.target_id is arista.avd.defined %}
+{%                         set target_id = transport.target.target_id | arista.avd.default('-') %}
+{%                     endif %}
+| {{ transport_name }} | {{ destination }} | {{ port }} | {{ gnmi_ssl_profile }} | {{ tunnel_ssl_profile }} | {{ vrf }} | {{ local_interface }} | {{ local_port }} | {{ target_id }} |
+{%                 endif %}
+{%             endfor %}
+{%         endif %}
+{%     endif %}
+
```

Comment:
Remove this line to not introduce extra blank lines in documentation

Reply:
This one was necessary to create spacing between the provider statement and the grpc-tunnel details.

Reply:
the line should be inside the `if` statement then (invert line 68 and 69)